### ISSUE TITLE
brew install grpc fails

### ISSUE THREAD

**Issue by Firefly35** [Homebrew/linuxbrew-core#21699]
# Bug report
## What you were trying to do (and why)

Install grpc on linux Mint with brew.

## What happened (include command output)
gpr dependency library link phase has failed

<details>
  <summary>Command output</summary>
  <pre>

  /home/linuxbrew/.linuxbrew/Cellar/cmake/3.19.1/bin/cmake -E cmake_link_script CMakeFiles/gpr.dir/link.txt --verbose=1
/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/shims/linux/super/g++-7 -fPIC -DNDEBUG -shared -Wl,-soname,libgpr.so.13 -o libgpr.so.13.0.0 CMakeFiles/gpr.dir/src/core/lib/gpr/alloc.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/atm.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/cpu_iphone.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/cpu_linux.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/cpu_posix.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/cpu_windows.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/env_linux.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/env_posix.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/env_windows.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/log.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/log_android.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/log_linux.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/log_posix.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/log_windows.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/murmur_hash.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/string.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/string_posix.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/string_util_windows.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/string_windows.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/sync.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/sync_abseil.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/sync_posix.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/sync_windows.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/time.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/time_posix.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/time_precise.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/time_windows.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/tls_pthread.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/tmpfile_msys.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/tmpfile_posix.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/tmpfile_windows.cc.o CMakeFiles/gpr.dir/src/core/lib/gpr/wrap_memcpy.cc.o CMakeFiles/gpr.dir/src/core/lib/gprpp/arena.cc.o CMakeFiles/gpr.dir/src/core/lib/gprpp/fork.cc.o CMakeFiles/gpr.dir/src/core/lib/gprpp/global_config_env.cc.o CMakeFiles/gpr.dir/src/core/lib/gprpp/host_port.cc.o CMakeFiles/gpr.dir/src/core/lib/gprpp/mpscq.cc.o CMakeFiles/gpr.dir/src/core/lib/gprpp/thd_posix.cc.o CMakeFiles/gpr.dir/src/core/lib/gprpp/thd_windows.cc.o CMakeFiles/gpr.dir/src/core/lib/profiling/basic_timers.cc.o CMakeFiles/gpr.dir/src/core/lib/profiling/stap_timers.cc.o  -ldl -lrt -lm -lpthread /home/linuxbrew/.linuxbrew/lib/libabsl_time.a /home/linuxbrew/.linuxbrew/lib/libabsl_synchronization.a /home/linuxbrew/.linuxbrew/lib/libabsl_strings.a /home/linuxbrew/.linuxbrew/lib/libabsl_base.a /home/linuxbrew/.linuxbrew/lib/libabsl_graphcycles_internal.a /home/linuxbrew/.linuxbrew/lib/libabsl_time.a /home/linuxbrew/.linuxbrew/lib/libabsl_civil_time.a /home/linuxbrew/.linuxbrew/lib/libabsl_time_zone.a /home/linuxbrew/.linuxbrew/lib/libabsl_stacktrace.a /home/linuxbrew/.linuxbrew/lib/libabsl_symbolize.a /home/linuxbrew/.linuxbrew/lib/libabsl_malloc_internal.a /home/linuxbrew/.linuxbrew/lib/libabsl_debugging_internal.a /home/linuxbrew/.linuxbrew/lib/libabsl_demangle_internal.a /home/linuxbrew/.linuxbrew/lib/libabsl_str_format_internal.a /home/linuxbrew/.linuxbrew/lib/libabsl_strings.a /home/linuxbrew/.linuxbrew/lib/libabsl_base.a -lpthread /home/linuxbrew/.linuxbrew/lib/libabsl_spinlock_wait.a -lrt /home/linuxbrew/.linuxbrew/lib/libabsl_strings_internal.a /home/linuxbrew/.linuxbrew/lib/libabsl_int128.a /home/linuxbrew/.linuxbrew/lib/libabsl_throw_delegate.a /home/linuxbrew/.linuxbrew/lib/libabsl_raw_logging_internal.a /home/linuxbrew/.linuxbrew/lib/libabsl_log_severity.a 
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_time.a(time.cc.o): relocation R_X86_64_32S against `.rodata' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_time.a(duration.cc.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_synchronization.a(mutex.cc.o): relocation R_X86_64_32 against `.bss' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_synchronization.a(create_thread_identity.cc.o): relocation R_X86_64_32 against `.bss' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_synchronization.a(waiter.cc.o): relocation R_X86_64_32 against `.rodata.str1.8' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_strings.a(str_cat.cc.o): relocation R_X86_64_32S against symbol `_ZN4absl14lts_2020_09_2316numbers_internal9kHexTableE' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_strings.a(numbers.cc.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_strings.a(ascii.cc.o): relocation R_X86_64_32S against symbol `_ZN4absl14lts_2020_09_2314ascii_internal8kToLowerE' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_strings.a(charconv.cc.o): relocation R_X86_64_32S against symbol `_ZN4absl14lts_2020_09_2316strings_internal10kFiveToNthE' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_strings.a(charconv_bigint.cc.o): relocation R_X86_64_32S against symbol `_ZN4absl14lts_2020_09_2316strings_internal10kFiveToNthE' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_strings.a(charconv_parse.cc.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_strings.a(memutil.cc.o): relocation R_X86_64_32S against symbol `_ZN4absl14lts_2020_09_2314ascii_internal8kToLowerE' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_strings.a(match.cc.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_base.a(spinlock.cc.o): relocation R_X86_64_32 against `.rodata' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_base.a(sysinfo.cc.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_base.a(thread_identity.cc.o): relocation R_X86_64_32 against `.bss' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_time.a(clock.cc.o): relocation R_X86_64_32 against `.bss' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_time_zone.a(time_zone_lookup.cc.o): relocation R_X86_64_32 against `.rodata.str1.8' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_time_zone.a(time_zone_fixed.cc.o): relocation R_X86_64_32 against `.rodata' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_time_zone.a(time_zone_impl.cc.o): relocation R_X86_64_32 against `.bss' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_time_zone.a(time_zone_if.cc.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_time_zone.a(time_zone_info.cc.o): relocation R_X86_64_32S against `.rodata' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_time_zone.a(time_zone_libc.cc.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_time_zone.a(time_zone_posix.cc.o): relocation R_X86_64_32S against `.rodata' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_stacktrace.a(stacktrace.cc.o): relocation R_X86_64_32 against `.text' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_symbolize.a(symbolize.cc.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_malloc_internal.a(low_level_alloc.cc.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_debugging_internal.a(address_is_readable.cc.o): relocation R_X86_64_32 against `.rodata.str1.8' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_debugging_internal.a(vdso_support.cc.o): relocation R_X86_64_32S against symbol `_ZN4absl14lts_2020_09_2318debugging_internal11ElfMemImage20kInvalidBaseSentinelE' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_debugging_internal.a(elf_mem_image.cc.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_demangle_internal.a(demangle.cc.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_str_format_internal.a(arg.cc.o): relocation R_X86_64_32S against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_str_format_internal.a(bind.cc.o): relocation R_X86_64_32 against `.rodata.str1.8' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_str_format_internal.a(extension.cc.o): relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_str_format_internal.a(float_conversion.cc.o): relocation R_X86_64_32 against `.bss' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_str_format_internal.a(output.cc.o): relocation R_X86_64_32 against symbol `__gxx_personality_v0@@CXXABI_1.3' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_str_format_internal.a(parser.cc.o): relocation R_X86_64_32S against `.rodata' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_int128.a(int128.cc.o): relocation R_X86_64_32 against `.rodata.str1.8' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_throw_delegate.a(throw_delegate.cc.o): relocation R_X86_64_32 against symbol `_ZNSt15underflow_errorD1Ev@@GLIBCXX_3.4' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /home/linuxbrew/.linuxbrew/lib/libabsl_raw_logging_internal.a(raw_logging.cc.o): relocation R_X86_64_32S against symbol `_ZN4absl14lts_2020_09_2313base_internal10AtomicHookIPFbNS0_11LogSeverityEPKciPPcPiEE13DummyFunctionES3_S5_iS7_S8_' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: final link failed: Nonrepresentable section on output
collect2: error: ld returned 1 exit status
CMakeFiles/gpr.dir/build.make:727: recipe for target 'libgpr.so.13.0.0' failed
make[2]: *** [libgpr.so.13.0.0] Error 1
make[2]: Leaving directory '/tmp/grpc-20201204-5279-160my7i/cmake/build'
CMakeFiles/Makefile2:336: recipe for target 'CMakeFiles/gpr.dir/all' failed
make[1]: *** [CMakeFiles/gpr.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[  7%] Building CXX object CMakeFiles/grpc_plugin_support.dir/src/compiler/python_generator.cc.o
/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/shims/linux/super/g++-7 -Dgrpc_plugin_support_EXPORTS -I/tmp/grpc-20201204-5279-160my7i/include -I/tmp/grpc-20201204-5279-160my7i -I/tmp/grpc-20201204-5279-160my7i/third_party/address_sorting/include -I/tmp/grpc-20201204-5279-160my7i/src/core/ext/upb-generated -I/tmp/grpc-20201204-5279-160my7i/src/core/ext/upbdefs-generated -I/tmp/grpc-20201204-5279-160my7i/third_party/upb -I/tmp/grpc-20201204-5279-160my7i/cmake/build/gens -DNDEBUG -fPIC -std=c++1z -o CMakeFiles/grpc_plugin_support.dir/src/compiler/python_generator.cc.o -c /tmp/grpc-20201204-5279-160my7i/src/compiler/python_generator.cc
[  7%] Building CXX object CMakeFiles/grpc_plugin_support.dir/src/compiler/ruby_generator.cc.o
/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/shims/linux/super/g++-7 -Dgrpc_plugin_support_EXPORTS -I/tmp/grpc-20201204-5279-160my7i/include -I/tmp/grpc-20201204-5279-160my7i -I/tmp/grpc-20201204-5279-160my7i/third_party/address_sorting/include -I/tmp/grpc-20201204-5279-160my7i/src/core/ext/upb-generated -I/tmp/grpc-20201204-5279-160my7i/src/core/ext/upbdefs-generated -I/tmp/grpc-20201204-5279-160my7i/third_party/upb -I/tmp/grpc-20201204-5279-160my7i/cmake/build/gens -DNDEBUG -fPIC -std=c++1z -o CMakeFiles/grpc_plugin_support.dir/src/compiler/ruby_generator.cc.o -c /tmp/grpc-20201204-5279-160my7i/src/compiler/ruby_generator.cc
[  7%] Linking CXX shared library libgrpc_plugin_support.so
/home/linuxbrew/.linuxbrew/Cellar/cmake/3.19.1/bin/cmake -E cmake_link_script CMakeFiles/grpc_plugin_support.dir/link.txt --verbose=1
/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/shims/linux/super/g++-7 -fPIC -DNDEBUG -shared -Wl,-soname,libgrpc_plugin_support.so.1 -o libgrpc_plugin_support.so.1.33.2 CMakeFiles/grpc_plugin_support.dir/src/compiler/cpp_generator.cc.o CMakeFiles/grpc_plugin_support.dir/src/compiler/csharp_generator.cc.o CMakeFiles/grpc_plugin_support.dir/src/compiler/node_generator.cc.o CMakeFiles/grpc_plugin_support.dir/src/compiler/objective_c_generator.cc.o CMakeFiles/grpc_plugin_support.dir/src/compiler/php_generator.cc.o CMakeFiles/grpc_plugin_support.dir/src/compiler/python_generator.cc.o CMakeFiles/grpc_plugin_support.dir/src/compiler/ruby_generator.cc.o  /home/linuxbrew/.linuxbrew/lib/libprotoc.so /home/linuxbrew/.linuxbrew/lib/libprotobuf.so -ldl -lrt -lm -lpthread -lpthread 
/home/linuxbrew/.linuxbrew/Cellar/cmake/3.19.1/bin/cmake -E cmake_symlink_library libgrpc_plugin_support.so.1.33.2 libgrpc_plugin_support.so.1 libgrpc_plugin_support.so
make[2]: Leaving directory '/tmp/grpc-20201204-5279-160my7i/cmake/build'
[  7%] Built target grpc_plugin_support
make[1]: Leaving directory '/tmp/grpc-20201204-5279-160my7i/cmake/build'
Makefile:151: recipe for target 'all' failed
make: *** [all] Error 2

==> Formula
Path: /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/grpc.rb
==> Configuration
HOMEBREW_VERSION: 2.6.0-76-gcb38fe6
ORIGIN: https://github.com/Homebrew/brew
HEAD: cb38fe6e3ff18f2515661e4bba0e35a670f4888a
Last commit: 10 hours ago
Core tap ORIGIN: https://github.com/Homebrew/linuxbrew-core
Core tap HEAD: b725a0ba7ea4595cdbc5b0300e821b0550518863
Core tap last commit: 6 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /home/linuxbrew/.linuxbrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: :0.0
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 2.6.3 => /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.3_2/bin/ruby
CPU: quad-core 64-bit haswell
Clang: N/A
Git: 2.17.1 => /usr/bin/git
Curl: 7.58.0 => /usr/bin/curl
Kernel: Linux 4.15.0-54-generic x86_64 GNU/Linux
OS: Linux Mint 19.2 Tina (tina)
Host glibc: 2.27
/usr/bin/gcc: 7.4.0
/usr/bin/ruby: N/A
glibc: N/A
gcc: N/A
xorg: N/A
==> ENV
HOMEBREW_CC: gcc-7
HOMEBREW_CXX: g++-7
MAKEFLAGS: -j4
CMAKE_PREFIX_PATH: /home/linuxbrew/.linuxbrew
PKG_CONFIG_PATH: /home/linuxbrew/.linuxbrew/opt/c-ares/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/gflags/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/openssl@1.1/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/protobuf/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/re2/lib/pkgconfig
HOMEBREW_GIT: git
ACLOCAL_PATH: /home/linuxbrew/.linuxbrew/share/aclocal
PATH: /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/shims/linux/super:/home/linuxbrew/.linuxbrew/opt/autoconf/bin:/home/linuxbrew/.linuxbrew/opt/automake/bin:/home/linuxbrew/.linuxbrew/opt/cmake/bin:/home/linuxbrew/.linuxbrew/opt/libtool/bin:/home/linuxbrew/.linuxbrew/opt/c-ares/bin:/home/linuxbrew/.linuxbrew/opt/gflags/bin:/home/linuxbrew/.linuxbrew/opt/openssl@1.1/bin:/home/linuxbrew/.linuxbrew/opt/protobuf/bin:/usr/bin:/bin:/usr/sbin:/sbin

Error: grpc 1.33.2 did not build
Logs:
     /home/ltouraine/.cache/Homebrew/Logs/grpc/02.make.cc
     /home/ltouraine/.cache/Homebrew/Logs/grpc/01.cmake
     /home/ltouraine/.cache/Homebrew/Logs/grpc/01.cmake.cc
     /home/ltouraine/.cache/Homebrew/Logs/grpc/02.make
     /home/ltouraine/.cache/Homebrew/Logs/grpc/00.options.out
READ THIS: https://docs.brew.sh/Troubleshooting

  </pre>
</details>

## What you expected to happen

I expected grpc to build out of the box.

## Step-by-step reproduction instructions (by running `brew install` commands)

brew install grpc --verbose

**Comment by Almenon** [comment]
Same error w/ WSL2:

```
==> Installing grpc
==> cmake ../.. -DCMAKE_CXX_STANDARD=17 -DCMAKE_CXX_STANDARD_REQUIRED=TRUE -DBUILD_SHARED_LIBS=ON -DgRPC_BUILD_TESTS=OFF -DgRPC_INSTALL=ON -DgRPC_ABSL_PROVIDER=package -DgRPC_CA
==> make install
Last 15 lines from /home/almenon/.cache/Homebrew/Logs/grpc/02.make:
make[2]: Leaving directory '/tmp/grpc-20201206-19318-9nqawh/cmake/build'
make[1]: *** [CMakeFiles/Makefile2:208: CMakeFiles/gpr.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[  7%] Building CXX object CMakeFiles/grpc_plugin_support.dir/src/compiler/python_generator.cc.o
/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/shims/linux/super/g++-9 -Dgrpc_plugin_support_EXPORTS -I/tmp/grpc-20201206-19318-9nqawh/include -I/tmp/grpc-20201206-19318-9nqawh -I/tmp/grpc-20201206-19318-9nqawh/third_party/address_sorting/include -I/tmp/grpc-20201206-19318-9nqawh/src/core/ext/upb-generated -I/tmp/grpc-20201206-19318-9nqawh/src/core/ext/upbdefs-generated -I/tmp/grpc-20201206-19318-9nqawh/third_party/upb -I/tmp/grpc-20201206-19318-9nqawh/cmake/build/gens -DNDEBUG -fPIC -std=c++17 -o CMakeFiles/grpc_plugin_support.dir/src/compiler/python_generator.cc.o -c /tmp/grpc-20201206-19318-9nqawh/src/compiler/python_generator.cc
[  7%] Building CXX object CMakeFiles/grpc_plugin_support.dir/src/compiler/ruby_generator.cc.o
/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/shims/linux/super/g++-9 -Dgrpc_plugin_support_EXPORTS -I/tmp/grpc-20201206-19318-9nqawh/include -I/tmp/grpc-20201206-19318-9nqawh -I/tmp/grpc-20201206-19318-9nqawh/third_party/address_sorting/include -I/tmp/grpc-20201206-19318-9nqawh/src/core/ext/upb-generated -I/tmp/grpc-20201206-19318-9nqawh/src/core/ext/upbdefs-generated -I/tmp/grpc-20201206-19318-9nqawh/third_party/upb -I/tmp/grpc-20201206-19318-9nqawh/cmake/build/gens -DNDEBUG -fPIC -std=c++17 -o CMakeFiles/grpc_plugin_support.dir/src/compiler/ruby_generator.cc.o -c /tmp/grpc-20201206-19318-9nqawh/src/compiler/ruby_generator.cc
[  7%] Linking CXX shared library libgrpc_plugin_support.so
/home/linuxbrew/.linuxbrew/Cellar/cmake/3.19.1/bin/cmake -E cmake_link_script CMakeFiles/grpc_plugin_support.dir/link.txt --verbose=1
/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/shims/linux/super/g++-9 -fPIC -DNDEBUG -shared -Wl,-soname,libgrpc_plugin_support.so.1 -o libgrpc_plugin_support.so.1.33.2 CMakeFiles/grpc_plugin_support.dir/src/compiler/cpp_generator.cc.o CMakeFiles/grpc_plugin_support.dir/src/compiler/csharp_generator.cc.o CMakeFiles/grpc_plugin_support.dir/src/compiler/node_generator.cc.o CMakeFiles/grpc_plugin_support.dir/src/compiler/objective_c_generator.cc.o CMakeFiles/grpc_plugin_support.dir/src/compiler/php_generator.cc.o CMakeFiles/grpc_plugin_support.dir/src/compiler/python_generator.cc.o CMakeFiles/grpc_plugin_support.dir/src/compiler/ruby_generator.cc.o  /home/linuxbrew/.linuxbrew/lib/libprotoc.so /home/linuxbrew/.linuxbrew/lib/libprotobuf.so -ldl -lrt -lm -lpthread -lpthread
/home/linuxbrew/.linuxbrew/Cellar/cmake/3.19.1/bin/cmake -E cmake_symlink_library libgrpc_plugin_support.so.1.33.2 libgrpc_plugin_support.so.1 libgrpc_plugin_support.so
make[2]: Leaving directory '/tmp/grpc-20201206-19318-9nqawh/cmake/build'
[  7%] Built target grpc_plugin_support
make[1]: Leaving directory '/tmp/grpc-20201206-19318-9nqawh/cmake/build'
make: *** [Makefile:152: all] Error 2

READ THIS: https://docs.brew.sh/Troubleshooting

These open issues may also help:
brew install grpc fails https://github.com/Homebrew/linuxbrew-core/issues/21699
almenon@DESKTOP-QAK2P0K:~/couchers$ brew doctor
Your system is ready to brew.
```

**Comment by iMichka** [comment]
Fixed by #21727
